vault/index.html

```diff
@@ -17,81 +17,54 @@
-            </div>
-            <div class="card-footer">
-                <small class="text-muted">Last updated April 28, 2019</small> <!-- Make sure you Update! -->
-                <span style="float:right;">
-                    <a tabindex="0" data-toggle="popover" data-trigger="focus" title="Information" data-content="Sorry, we're currently developing this page!">Read More</a>
-                </span>
-            </div>
-        </div>
-        <div class="card">
-            <a href="#">
-                <img src="assets/bjncloud/bjncloud-logo.png" class="card-img-top" alt="Logo of BJNCloud Hosting">
-                <div class="card-body">
-                    <h5 class="card-title">BJNCloud Hosting</h5>
-            </a>
-            <p class="card-text">Host ran by a notirious active leaker JesusX0 (a.k.a NiñoRataPistolita on BlackShitMC). They are also responsible for making a clone of an old host named Dedifire and claim that the host was sold to them.</p>
-        </div>
-        <div class="card-footer">
-            <small class="text-muted">Last updated N/A</small>
-            <span style="float:right;">
-                <a tabindex="0" data-toggle="popover" data-trigger="focus" title="Information" data-content="Sorry, we're currently developing this page!">Read More</a>
-            </span>
-        </div>
-    </div>
-    <div class="card">
+           
+             <div class="card">
         <a href="#">
-            <img src="assets/mcshostingltd/mcshostingltd-logo.png" class="card-img-top" alt="Logo of MCSHosting LTD">
+            <img src="assets/itzawesome/itzawesome.png" class="card-img-top" alt="Avatar of Voxtility from MCM">
             <div class="card-body">
-                <h5 class="card-title">MCS Hosting LTD</h5>
+                <h5 class="card-title">ItzAwesome</h5>
         </a>
-        <p class="card-text">Ddosing competitors (Deploys.io - then DeluxeNode). Nicke also lied to me that it was false accusations in an attempt to get his horrible company removed from the site. As soon as he saw the proof, he cried 'It has been paid for'.</p>
+        <p class="card-text">This unintelligent leaker paid for a resource using his leak name (awecrift). While attempting to crack the resource, he removed one too many lines and resulted in nuking one of his nodes. 'I won't leak anymore'[sic]</p>
     </div>
     <div class="card-footer">
         <small class="text-muted">Last updated N/A</small>
         <span style="float:right;">
             <a tabindex="0" data-toggle="popover" data-trigger="focus" title="Information" data-content="Sorry, we're currently developing this page!">Read More</a>
         </span>
     </div>
-    </div>
+
```

Comment:
Forgot the `</div>` here. Which begs the question.. Where did drop the original end tag?

Reply:
Nevermind, seems you accidentally deleted it.
```suggestion
</div>
```